pkg/service/v0/service.go

```diff
@@ -115,14 +115,15 @@ func getFailsafeIdentifier(c context.Context, identifier *proto.Identifier) *pro
 		identifier = &proto.Identifier{}
 	}
 	if identifier.AccountUuid == "me" {
-		ownAccountUUID := c.Value(middleware.UUIDKey).(string)
-		if len(ownAccountUUID) > 0 {
-			identifier.AccountUuid = ownAccountUUID
-		} else {
-			// might be valid for the request not having an AccountUuid in the identifier.
-			// but clear it, instead of passing on `me`.
-			identifier.AccountUuid = ""
+		ownAccountUUID, ok := c.Value(middleware.UUIDKey).(string)
+		if ok {
+			if len(ownAccountUUID) > 0 {
+				identifier.AccountUuid = ownAccountUUID
+			}
 		}
+		// might be valid for the request not having an AccountUuid in the identifier.
+		// but clear it, instead of passing on `me`.
+		identifier.AccountUuid = ""
```

Comment:
Doesn't this overwrite what's changed above if the if block. I think It should go in the else block :thinking:

Reply:
True, there should be a `return` statement after line 121

Reply:
actually, it's shorter than that

Reply:
@kulmann can you check the semantics remain untouched with the latest commit?